### ISSUE TITLE
Ic-light model don't appear and work on the node

### ISSUE THREAD

**Issue by rongtian0927** [huagetai/ComfyUI-Gaffer#4]
![image](https://github.com/huagetai/ComfyUI-Gaffer/assets/123718340/b1e520cd-ad05-4e1d-9358-c947f0c5796b)
I have follow you intstruction put the model at model file with file name (iclight)

**Comment by huagetai** [comment]
![5](https://github.com/huagetai/ComfyUI-Gaffer/assets/1137341/daded052-0fe4-4ddf-ba9f-ad7cb50fb12d)
Are you sure the model files are placed in this path?

**Comment by rongtian0927** [comment]
> ![5](https://private-user-images.githubusercontent.com/1137341/333807825-daded052-0fe4-4ddf-ba9f-ad7cb50fb12d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY3MDYxMzAsIm5iZiI6MTcxNjcwNTgzMCwicGF0aCI6Ii8xMTM3MzQxLzMzMzgwNzgyNS1kYWRlZDA1Mi0wZmU0LTRkZGYtYmE5Zi1hZDdjYjUwZmIxMmQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjZUMDY0MzUwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NWZjZTA4YjI4NWUyODhhMDcwZmVmMzYxNjQ1ZDgzNWFlZDM3MTIwYjljMDRhZDdjODA5YzQxYWFkYTI5MDk1YSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.t5Ap15qDJrvOEddFqQEv4eCbP9naJR4e4UIGC20ps7E) Are you sure the model files are placed in this path?

yes I am sure that it is inside the model and iclight file, I am using comfyui by rundiffusion.
